Assets/Scripts/Game/FadeBehaviour.cs

```diff
@@ -64,10 +64,11 @@ public void FadeHUDToBlack(float fadeDuration = 0.5f)
             if (fadeTargetPanel == null || !allowFade)
                 return;
 
-            fadeStartColor = Color.clear;
+            fadeStartColor = new Color(0.39f, 0, 0, 0.4f);
```

Comment:
I'd like this change not to be made, thank you. The entire purpose of FadeHUDToBlack() is to fade from clear to black. This changes the purpose of the method and hardcodes a different start colour. It is used in many more places than just when player dies.

Reply:
I have an alternative solution, then.  But by the way, visual studio's "Find all references" when used on FadeHUDToBlack() yields only one occurrence in PlayerDeath.  But regardless it's best to keep this method unchanged in case other code uses it.

Regardless, The better solution is that I can create another method FadeHUDBloodToBlack() in FadeBehavior.cs and have PlayerDeath call that method instead.